### DISCUSSION TITLE
Why webgpu is not released in chrome-android while it works nicely in chrome-canary-android  for months ?

### DISCUSSION THREAD

**Discussion by tlecoz** [gpuweb/gpuweb#4101] [Q&A] [answered]
I feel so frustrated .... 😭

I cannot find any information about a release date for chrome ios & android , can you share some information about this  ?

**Comment by kainino0x** [comment, marked as answer]
WebGPU in Chrome works well on a lot of Android phones, especially newer/higher-end ones, but we still have some gaps before it's reliable enough to ship to the rest of Android devices in the world. In particular, we need to do more automated testing on multiple Android GPU vendors and identify and fix major compatibility issues before we can ship. Unfortunately I can't offer a very concrete estimate yet but we hope to have it... sometime this year.

Chrome on iOS does not use the Chromium engine, it uses Safari under the hood, so will ship WebGPU when Safari does.

**Comment by tlecoz** [comment]
Thank you for your message ! 
"we hope to have it... sometime this year."
I hope so too ! :) 

"will ship WebGPU when Safari does." 
sometime this year ? 😇
